GSoC-2021/Anoushka-Ramesh/README.md

```diff
@@ -0,0 +1,88 @@
+# GSoC2021 Expression of Interest: Anoushka Ramesh
+Email - anoushkaramesh21@gmail.com
+
+## Assessment Step 1
+Website - [https://gsocautspaces.herokuapp.com](https://gsocautspaces.herokuapp.com)
+
+Activity page - [https://www.openhumans.org/activity/project-autspaces/#activity-panel-data](https://www.openhumans.org/activity/project-autspaces/#activity-panel-data)
+
+UserData JSON file - 
+![Screenshot from 2021-03-29 22-59-34](https://user-images.githubusercontent.com/73518403/112876035-a0388780-90e2-11eb-8e98-fd2a71a55d11.png)
+
+## Assessment Step 2
+[Todo-App](https://github.com/anoura12/todo-app) - A basic Django todo application where you can create and update tasks.
```

Comment:
Looks good - do you have a deployment somewhere for me to test?

I note that the repository doesn't have a licence nor any installation instructions. You don't have to add those (I don't want you to do lots of work on a demo project) but if you have a repository that does have documentation please can you add that link too? It's ok if the documentation isn't for a Django app.

Reply:
Hi @KirstieJane,
I've made a new commit which includes the link to the deployed app for you to use it.
I've also added a license to my repository and I will be adding some installation instructions soon.